spi/common/core-spi/src/main/java/org/eclipse/dataspaceconnector/spi/types/domain/transfer/DataFlowRequest.java

```diff
@@ -37,6 +37,7 @@ public class DataFlowRequest implements Polymorphic, TraceCarrier {
     private DataAddress sourceDataAddress;
     private DataAddress destinationDataAddress;
 
+    private String callbackAddress;
```

Comment:
Could it be an URI/URL directly?

Reply:
Does Jackson support URL or do we have a custom ser/de for that?

Reply:
It should be supported as well as URI, just tried it out in an unit test